src/styles/blocks/article.scss

```diff
@@ -0,0 +1,28 @@
+.article {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: inline-block;
```

Comment:
check width of your image for 744 screen resolution. Image should  be smaller and title and text should have another width for this resolution

Reply:
resolved